src/utils/syncRelationships.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-param-reassign */
+import {
+  pushInverse,
+  put,
+  putMany,
+} from '../queries';
+
+export default (record, schema, id) => {
+  const syncRelationshipsArray = [];
+
+  Object.keys(schema.relationships).forEach(relationship => {
+    const relationshipObject = schema.relationships[relationship];
+    const { field, relation, inverse } = relationshipObject;
+    const { type: inverseType, field: inverseField, relation: inverseRelation } = inverse;
+
+    const data = record[field];
+
+    let queryToPush;
+
+    if (record.hasOwnProperty(field)) {
+      if (inverseRelation === 'hasMany') {
+        queryToPush = Array.isArray(data)
+          ? pushInverse(inverseType, inverseField, [], data, id)
+          : pushInverse(inverseType, inverseField, [], [data], id);
```

Comment:
Weird use of the ternary operator. The only thing that changes is encapsulating `data` in an array. I think it would be cleaner and more legible to change up the ternary operator or create a new variable, i.e.:

```
const dataCoercedToArray = Array.isArray(data) ? data : [data];
queryToPush = pushInverse(inverseType, inverseField, [], dataCoercedToArray, id);
```